l2geth/consensus/clique/scheduler.go

```diff
@@ -161,8 +175,14 @@ func (schedulerInst *Scheduler) AddPeerCheck() {
 }
 
 func (schedulerInst *Scheduler) schedulerRoutine() {
-	batchSize := uint64(10) // 10 transaction in one batch
-	expireTime := int64(15) // 15s
+	batchSize := uint64(100) // 10 transaction in one batch
+	expireTime := int64(60)  // 15s
```

Comment:
The comment is not accurate. Besides, please define to const value `defaultBatchSize`  and `defaultExpireTime` instead